libpod/container_internal_linux.go

```diff
@@ -496,6 +498,42 @@ func (c *Container) addNamespaceContainer(g *generate.Generator, ns LinuxNS, ctr
 	return nil
 }
 
+func (c *Container) exportCheckpoint(dest string) (err error) {
```

Comment:
We shouldn't allow export of anything with named volumes (`len(c.config.NamedVolumes) > 0`) - that could produce an inconsistent state on the remote side that could lead to being unable to remove the container.

Same for containers with dependencies (`len(c.Dependencies() > 0`).